### DISCUSSION TITLE
gmail smtp 邮箱配置不能正常工作

### DISCUSSION THREAD

**Discussion by jason-xie-123** [tryzealot/zealot#1543] [Q&A] [answered]
请教个问题, 不知有没有人遇见过, 我这边在 ubuntu ec2 上面通过 https://github.com/tryzealot/zealot-docker + nginx 配置好了 tryzealot/zealot, 一切工作正常, 但是我在配置 gmail 邮箱的时候一直不能正常工作, 我使用下面的配置设置

`
{
  "address": "smtp.gmail.com",
  "port": 587,
  "domain": "gmail.com",
  "username": "xxx@gmail.com",
  "password": "xxxxxxxx",
  "auth_method": "plain",
  "enable_starttls_auto": true
}
`

我在 https://www.gmass.co/smtp-test 里面测试的账号密码是可以正常工作的

**Comment by icyleaf** [comment, marked as answer]
确认是 bug 已在 #1547 修复，目前可用 nightly-2024-06-18 可用后续会发布 5.3.1，需要注意的是 SMTP 配置有所变化

```diff
SMTP_DOMAIN=zealot.com
SMTP_ADDRESS=smtp.gmail.com
SMTP_PORT=587
SMTP_USERNAME=you@gmail.com
SMTP_PASSWORD=your_app_password
SMTP_AUTH_METHOD=plain
-SMTP_ENABLE_STARTTLS_AUTO=true
+SMTP_ENABLE_STARTTLS=true
```
